### DISCUSSION TITLE
[FEATURE] Add changelog on UN1CA Updates

### DISCUSSION THREAD

**Discussion by gbfllp** [salvogiangri/UN1CA#203] [Feature requests]
If possible, add a changelog of the last update on UN1CA update menu

**Comment by salvogiangri** [comment]
This was meant to be a thing with the first release of the app, however I didn't make it due to lack of time and my will to get 2.5.0 released as fast as possible. Here's a mock up of the UI with the changelog page button in one of the earliest build of the app:

<img loading="lazy" src="https://github.com/user-attachments/assets/c0e237f8-e40e-4bd4-b71a-001f4143922a" width="25%"/>

**Comment by salvogiangri** [comment]
Better late than never I guess...😅

<img loading="lazy" src="https://github.com/user-attachments/assets/15f9f8f2-97c3-4923-9809-de879b16ea63" width="25%"/>
<img loading="lazy" src="https://github.com/user-attachments/assets/f28f8f6d-51e7-4f41-a7fd-c81a5dd3cb59" width="25%"/>
